docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -73,17 +74,72 @@ the light client isn’t expired.
+Each node will then verify the evidence according to the CometBFT rules that define a valid double signing infraction, and based on this verification, they will decide whether to add the evidence to a block. 
+During the evidence verification process, the signatures of the conflicting votes must be verified successfully. 
+Note that this is achieved using the public key of the misbehaving validator, along with the `ChainID` of the chain where the infraction occurred (see [CometBFT equivocation verification](https://github.com/cometbft/cometbft/blob/2af25aea6cfe6ac4ddac40ceddfb8c8eee17d0e6/spec/consensus/evidence.md#L95)).
+
+Once a double signing evidence is committed to a block, the consensus layer will report the equivocation to the evidence module of the Cosmos SDK application layer. 
+The application will, in turn, punish the malicious validator through jailing, tombstoning and slashing 
+(see [handleEquivocationEvidence](https://github.com/cosmos/cosmos-sdk/blob/19389505643500b25a5efeb02715c3deef9b6ede/x/evidence/keeper/infraction.go#L27C17-L27C43)).
+
+## Decision
+
+In the second part of the feature, we will introduce a new endpoint `HandleConsumerDoubleVoting(
+ctx sdk.Context, evidence *tmtypes.DuplicateVoteEvidence, chainID string, pubkey cryptotypes.PubKey)`. 
+Simply put, the handling logic will verify a double signing evidence against a provided 
+public key and chain ID and, if successful, execute the jailing of the malicious validator who double voted.
+  
+We will define a new 
+[`MsgSubmitConsumerDoubleVoting`](https://github.com/cosmos/interchain-security/blob/20b0e35a6d45111bd7bfeb6845417ba752c67c60/proto/interchain_security/ccv/provider/v1/tx.proto#L69C9-L69C38) 
+message to report a double voting evidence observed 
+on a consumer chain to the endpoint of the provider chain. This message will contain two fields: 
+a double signing evidence 
+[`duplicate_vote_evidence`](https://github.com/cosmos/interchain-security/blob/20b0e35a6d45111bd7bfeb6845417ba752c67c60/proto/interchain_security/ccv/provider/v1/tx.proto#L75) 
+and a light client header for the infraction block height, 
+referred to as [`infraction_block_header`](https://github.com/cosmos/interchain-security/blob/20b0e35a6d45111bd7bfeb6845417ba752c67c60/proto/interchain_security/ccv/provider/v1/txproto#L77). 
+The latter will provide the malicious validator's public key and the chain ID required to verify the signature of the votes contained in the evidence.
+ 
+Note that double signing evidence will be verified by using the same conditions than in the CometBFT 
+[`verify(evidence types.Evidence)`](https://github.com/cometbft/cometbft/blob/2af25aea6cfe6ac4ddac40ceddfb8c8eee17d0e6/evidence/verify.go#L19) 
+method, with the exception that its age won't be checked. 
+This is primarily because, for the first stage of this feature, 
+the goal is to jail validators for double signing only. Since the jail time doesn't depend on the evidence's age, 
```

Comment:
I didn't get why you mention "jail time doesn't depend on evidence age => it can be ignored" - if we would e.g. slash would the evidence age matter more?

Reply:
The technical limitations, on a quick read, also don't mention the evidence age, just infraction height. Something seems missing in the reasoning to me.

Reply:
The evidence age is checked because it should be less than the unbonding period so the validators still can be punished, i.e. still bonded. 

> if we would e.g. slash would the evidence age matter more?
That's correct. The evidence age check ensures that if a slashing execution happens all the unbonding delegations that participated in the infraction stake are slashed.

On the opposite, the jailing time for double signing is set to a maximum duration, independently of the infraction height, and only bonded validators can get jailed. In consequence, removing the evidence age check doesn't change the jailing execution.

Reply:
But removing the evidence age changes the behavior. A validator may get jailed due to some "old" evidence on a consumer, while it wouldn't get jailed if the consumer had been a standalone chain.